### PR TITLE
enable undockerized Wagtail development

### PR DESCRIPTION
**Pull request by Niicck**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+psycopg2-binary = "==2.7.4"
+django-webpack-loader = "==0.6.0"
+boto3 = "==1.9.94"
+django-storages = "==1.6.6"
+django-dbbackup = "==3.2.0"
+django-smuggler = "==0.9.1"
+PyYAML = ">=4.2b1"
+
+[requires]
+python_version = "3.7"
```
Comment (easherma) [inline Pipfile]: I'm down for this(and its the version i use locally), but FYI we had python version pinned to 3.6 in our Dockerfiles

File: README.md [hunk 2]
```diff
@@ -80,6 +80,15 @@ HARD_REBUILD="on" ./scripts/serve-local.sh
 -   It takes 90 seconds to do a HARD_REBUILD.
 -   If worse comes to worse, you can always delete your local joplin docker images with `docker rmi`.
 
+**Run undockerized**
+
+You might prefer to run the Django app on your host computer to enable better access to debugging tools. This script will still run joplin_assets and joplin_db on docker containers, but will run a django `runserver` command directly on your host computer.
```
Comment (easherma) [inline README.md]: WHY YES I WOULD PREFER THANK U

File: scripts/serve-local.sh [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 set -o errexit
 
-CURRENT_DIR=`dirname $BASH_SOURCE`
-source $CURRENT_DIR/docker-helpers.sh
+D=`dirname $BASH_SOURCE`
```
Comment (easherma) [inline scripts/serve-local.sh]: I tend to prefer more explicit names, any significant reason to shorten this one?

File: scripts/undockered.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```
Comment (easherma) [inline scripts/undockered.sh]: https://www.imdb.com/title/tt1853728/ Django Undockered!

File: scripts/serve-local.sh (hunk 3 above)
Comment (Niicck) [reply scripts/serve-local.sh]: is long. I started volunteering on another django project and I switched my naming convention. But CURRENT_DIR isn't hurting anyone. It stays.

File: Pipfile (hunk 1 above)
Comment (Niicck) [reply Pipfile]: Good call. I'd vote in favor of us moving Joplin to the latest version of Joplin.

Comment (easherma) [reply Pipfile]: @briaguya should we just up the python version in our docker containers and see if we notice anything breaking or hold off for now?

**Pull request merged**